dandalandone/POC-EyCommerce
# [API Integration] - Delete Cart Item button in Cart Page

## Ticket/s

- [Delete Cart Item button](https://trello.com/c/Jcol4sET/87-delete-cart-item)

## Changes

- implement Delete button in Cart Page

## Screenshot

![image](https://github.com/dandalandone/POC-EyCommerce/assets/129832182/b154abb7-42f5-4f1c-b1b3-b01e5124ac91)

![image](https://github.com/dandalandone/POC-EyCommerce/assets/129832182/e6bc9f70-d071-41cf-adf4-7c254780d6b0)


## How to test?

- run `pnpm run dev`
- go to http://localhost:5173/cart
- make sure you have item in your Cart
- press the `delete icon`
